src/molecules/checkbox-group/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { CSSProperties } from 'react';
+import Checkbox from '../../atoms/checkbox';
+import ErrorField from '../../atoms/error-field';
+import * as Styles from './styles';
+
+export type CheckboxItem = {
+  value: string;
+  label?: string | React.ReactNode;
+  checked?: boolean;
+  onChange?: () => {};
```

Comment:
I guess the type should be closer to `onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;` but it'd require changes also in `src/atoms/checkbox/index.tsx` so I don't insist on any changes